dsse/dsse.go

```diff
@@ -61,12 +54,20 @@ type Envelope struct {
 }
 
 type Signature struct {
-	KeyID         string   `json:"keyid"`
-	Signature     []byte   `json:"sig"`
-	Certificate   []byte   `json:"certificate,omitempty"`
-	Intermediates [][]byte `json:"intermediates,omitempty"`
+	KeyID         string               `json:"keyid"`
+	Signature     []byte               `json:"sig"`
+	Certificate   []byte               `json:"certificate,omitempty"`
+	Intermediates [][]byte             `json:"intermediates,omitempty"`
+	Timestamps    []SignatureTimestamp `json:"timestamps,omitempty"`
```

Comment:
This looks like a timestamp signature on the signature, correct?

Reply:
Correct